openhands/cli/settings.py

```diff
@@ -397,18 +503,28 @@ async def modify_llm_settings_advanced(
             session,
             '(Step 1/6) Custom Model (CTRL-c to cancel): ',
             error_message='Custom Model cannot be empty',
+            default_value=llm_config.model or '',
         )
 
         base_url = await get_validated_input(
             session,
             '(Step 2/6) Base URL (CTRL-c to cancel): ',
             error_message='Base URL cannot be empty',
+            default_value=llm_config.base_url or '',
         )
 
+        prompt_text = '(Step 3/6) API Key (CTRL-c to cancel): '
+        current_api_key = (
+            llm_config.api_key.get_secret_value() if llm_config.api_key else ''
+        )
+        if current_api_key:
+            prompt_text = '(Step 3/6) API Key [***] (CTRL-c to cancel, ENTER to keep current, type new to change): '
```

Comment:
We can probably show the first and last bit of the API Key for information
```suggestion
            prompt_text = '(Step 3/6) API Key [{current_api_key[:4]}***{current_api_key[-4:]}] (CTRL-c to cancel, ENTER to keep current, type new to change): '
```

Reply:
I see. I will fix it and update the test